src/docfx/lib/JsonUtility.cs

```diff
@@ -94,6 +92,15 @@ public static T Deserialize<T>(string json)
             return Deserialize<T>(new StringReader(json));
         }
 
+        /// <summary>
+        /// Parse a string to JToken.
+        /// Validate null value during the process.
+        /// </summary>
+        public static (List<Error>, JToken) Parse(string json)
```

Comment:
>Parse [](start = 44, length = 5)

Instead of having a new `Parse` method, modify the above 2 `Deserialize` methods to return `(errors, T, sourceMap)`, the idea is for JSON and YAML to share the same API signature.

You could implement JSON source map later (by using JsonReader API)

Reply:
fixed